django/middleware/csrf.py

```diff
@@ -151,7 +151,8 @@ def _accept(self, request):
         return None
 
     def _reject(self, request, reason):
-        logger.warning(
+        security_logger = logging.getLogger('django.security.csrf')
```

Comment:
I don't think we need the `django.request` logger in this module anymore; why not get this logger at module level instead, rather than every time this method is called?